app/controllers/journals_controller.rb

```diff
@@ -110,17 +84,6 @@ def diff
     end
   end
 
-  def preview
-    @journal.notes = params[:notes]
-
-    respond_to do |format|
-      format.any(:html, :js) do
-        render locals: { journal: @journal },
-               layout: false
-      end
-    end
-  end
-
   private
 
   def find_journal
```

Comment:
I think there are a couple of methods in here that can also be removed:
* find_journal
* valid_vield?
* default_breadcrumb
* valid_diff?

In addition to that, some of the before filters can also be removed as can the `require 'diff'`